### ISSUE TITLE
Auto-capitalize each sentence in a new post title

### ISSUE THREAD

**Issue by abluescarab** [dessalines/jerboa#498]
- [x] Did you check to see if this issue already exists?
- [x] Is this only a single feature request? Do not put multiple feature requests in one issue.
- [x] Is this a question or discussion? Don't use this, use https://lemmy.ml/c/jerboa

It would be a great accessibility feature to auto-capitalize the first word of each sentence in a post title (a toggle in the settings to enable/disable this option would work, too, if it's not desired default behavior).

**Comment by dessalines** [comment]
Why is this accessibility, I've never heard of this before. If it is it might as well apply to all content, as a setting.

**Comment by abluescarab** [comment]
Capitalizing the first word of a sentence is easier to read for visually impaired people like me since the letters are easier to differentiate for new sentences. Additionally, it takes extra time to hit the "shift" button on a phone keyboard, which can mean that people who are slower typists due to hand issues/visual impairment/etc. will have a harder time typing.

**Comment by twizmwazin** [comment]
I'm down to support accessibility features that can help more people use Jerboa and Lemmy. I'm a little confused what you're asking for though, please let me know which of these you would want (or if I'm missing it altogether):

1. An option to filter only post titles to capitalize the first letter of each word
2. An option to filter all text to capitalize the first letters of words
3. An option to automatically capitalize the first letter of every word when creating a post or comment

**Comment by abluescarab** [comment]
@twizmwazin Number 3 would be what I was referring to. Sorry about the confusion!

**Comment by dessalines** [comment]
This is a really strange request, and should be a keyboard type thing.
